src/components/moderation/moderation-common.ts

```diff
@@ -55,6 +55,15 @@ import { set_interval } from "../../utils/node.js";
 export const duration_regex = /(?:perm\b|(\d+)\s*([mhdwMys]))/;
 
 export const moderation_on_team_member_message: string = "Can't apply this moderation on team members";
+export const joke_responses: Array<string> = new Array<string>(
+    "You won't get off that easy! ;)",
+    "Try again next time lmao",
+    "Didn't work. Maybe a skill issue?",
+);
+
+function get_random_array_element(arr: any[]) {
```

Comment:
This might be better put in `src/utils/array.ts`

Reply:
Sounds good!